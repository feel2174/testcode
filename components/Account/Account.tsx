const Account = (props: { name: string; mbti: string; age?: number }) => {
  return (
    <div>
      <h2>{props.name}</h2>
      <p>{props.name}</p>
      <p>{props.age}</p>
    </div>
  );
};

export default Account;
