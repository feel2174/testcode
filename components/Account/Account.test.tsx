import Account from './Account';
import { render } from '@testing-library/react';

test('matches snapshot', () => {
  const utils = render(<Account name="너구리호박" mbti="INFP" age={30} />);
  expect(utils.container).toMatchSnapshot();
});
