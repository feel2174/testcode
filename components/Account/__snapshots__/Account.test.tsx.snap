// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="css-1cvyltp"
  >
    <h2>
      너구리호박
    </h2>
    <p>
      INFP
    </p>
    <p>
      30
    </p>
  </div>
</div>
`;
